scheduler/kubernetes/support.md

```diff
@@ -35,17 +35,18 @@ If the PVC creation is failing, this could be due the following reasons
 * A firewall/iptables rule for port 9001 is present on the hosts running px containers. This prevents the create volume call to come to the Portworx API server.
 * For Kubernetes versions 1.6.4 and before, Portworx may not running on the Kubernetes master/controller node.
 * For Kubernetes versions 1.6.5 and above, if you don't have Portworx running on the master/controller node, ensure that
-    * The `portworx-service` is running in the `kube-system` namespace.
+    * The `portworx-service` Kubernetes `Service` is running in the `kube-system` namespace.
     * You don't have any custom taints on the master node. Doing so will disallow kube-proxy to run on master and that will cause the `portworx-service` to fail to handle requests.
 * The StorageClass name specified might be incorrect.
-* Make sure you are running Kubernetes 1.6 and above. Kubernetes 1.5 does not have our native driver which is required for PVC creation.
 * Describe the PVC using `kubectl describe pvc <pvc-name>` and look at errors in the events section which might be causing failure of the PVC creation.
+* Make sure you are running Kubernetes 1.6 and above. Kubernetes 1.5 does not have our native driver which is required for PVC creation.
```

Comment:
We ave a statement elsewhere saying "if you use Kubernetes 1.5, click <here>" -- but, I'd recommend we cut out the support for it altogether (it's probably very broken by now).